resources/styles/components/cc-dashboard/index.less

```diff
@@ -1,66 +1,98 @@
-.dashboard-header-links {
-  .link {
-    color: @tutor-tertiary-light;
-    text-transform: capitalize;
-    margin-left: 10px;
-    &.btn {
-      padding: 0;
+cc-dashboard {
+  .column-legend {
+    span {
```

Comment:
Is this a `react-bootstrap` class? If so, should the selector contain `>` so the styling doesn't bleed?